### PR TITLE
initial commit of issue templates

### PR DESCRIPTION
**Pull request by jonchurch**
This PR draft should land on something like a staging branch, once one exists.

It creates three issue templates, `bug` and `question` each apply the equivalent label. `other` is a potential catch all.

I will PR the `config.yml` separately.

### CODE REVIEW

**Comment by dougwilson** [comment]
> This PR draft should land on something like a staging branch, once one exists.

Just to clarify: are you asking for a separate branch to be made in this repo you want to target besides just `master`?

File: .github/ISSUE_TEMPLATE/2bug_report.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: Bug Report
+about: Have you found a bug?
```
Comment (dougwilson) [inline .github/ISSUE_TEMPLATE/2bug_report.md:3]:
```suggestion
about: Do you believe you have found a bug?
```

I'm not strongly for either wording, just wanted to throw this change out to see any thoughts.

**Comment by jonchurch** [comment]
@dougwilson Yes that's the idea, target a feature branch so we can see proper diffs as we build it out. Then merge to master once we are ready to see it go live.

**Comment by jonchurch** [comment]
I don't actually have a strong opinion on the workflow re: staging branch. Just wanted to note that this "releases" once it's merged to master 👍

**Comment by wesleytodd** [comment]
For the question one, we should add this link: http://www.nodeslackers.com/

File: .github/ISSUE_TEMPLATE/2bug_report.md (hunk 1 above)
Comment (wesleytodd) [reply .github/ISSUE_TEMPLATE/2bug_report.md:3]: 👍 on this change.  If I were a user and was unsure, I would feel more comfortable choosing this than the more definitive version.

**Comment by jonchurch** [comment]
@wesleytodd I added a link to the nodeslackers invite form, and a link to the express-js channel

**Comment by jonchurch** [comment]
I think these are about ready to go, my only question based on what's here is if we should use a label besides `bug` for initial bug reports. It's a process question, if we had a `bug` and a `replicated bug` label that could help us differentiate.

**Comment by dougwilson** [comment]
I think `bug` is fine. We can always improve it if we find it confusing or need some extra process.

**Comment by wesleytodd** [comment]
@jonchurch I think we need to land this (and probably a fast follow with a PR template). Are you around to help or can I jump in and push a few changes here?

File: .github/ISSUE_TEMPLATE/1question.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+about: Seeking help or have a question about usage?
+labels: question
+---
+
+<!-- We can't debug your app for you, but you can ask questions and we will try to answer them.
+
+You can also ask a question in the Node.js Slack community's #express-js channel:
+
+* Slack invite form: http://www.nodeslackers.com/invite
+* #express-js channel: https://node-js.slack.com/archives/C0L827VV4
```
Comment (wesleytodd) [inline .github/ISSUE_TEMPLATE/1question.md:12]: We should change this to the OpenJS Slack.

File: .github/ISSUE_TEMPLATE/1question.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+It is super important that you paste in samples of your code (please avoid screenshots of code)!
```
Comment (wesleytodd) [inline .github/ISSUE_TEMPLATE/1question.md:14]:
```suggestion
It is super important that you paste in samples of your code (no screenshots of code, [use markdown](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks))!
```

I feel like we should auto close anything which is a screenshot of code. And by linking here we help educate folks who might not know.

**Review by wesleytodd — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/1question.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+name: Question
+about: Seeking help or have a question about usage?
+labels: question
+---
+
+<!-- We can't debug your app for you, but you can ask questions and we will try to answer them.
+
+You can also ask a question in the Node.js Slack community's #express-js channel:
+
+* Slack invite form: http://www.nodeslackers.com/invite
```
Comment (wesleytodd) [inline .github/ISSUE_TEMPLATE/1question.md:11]:
```suggestion
* Slack invite form: https://openjsf.org/collaboration
```

**Comment by wesleytodd** [comment]
I had second thoughts about the change to point at the openjs slack. Maybe it is better for us to keep the community channel and then have the openjs slack be more for collaborators and contributors to chat. Anyone opposed to me committing that suggestion and then merging this?

**Comment by UlisesGascon** [comment]
Sounds good @wesleytodd, I like the idea of keeping the Slack for collaborators/contributors.

**Pull request merged**